desc/compute/_geometry.py

```diff
@@ -176,27 +176,39 @@ def _A_of_z(params, transforms, profiles, data, **kwargs):
     label="A(\\zeta)",
     units="m^{2}",
     units_long="square meters",
-    description="Cross-sectional area as function of zeta",
+    description="Enclosed cross-sectional (constant phi surface) area, "
+    "as function of zeta",
     dim=1,
     params=[],
     transforms={"grid": []},
     profiles=[],
     coordinates="z",
-    data=["Z", "n_rho", "g_tt"],
+    data=["Z", "n_rho", "e_theta|r,p", "rho"],
```

Comment:
why this change? is this for when omega is nonzero?

Reply:
Yes, I was checking the math for this to be sure, and i was gonna need to do this anyway so I did it here